modules/core/src/transitions/transition.js

```diff
@@ -76,7 +77,6 @@ export default class Transition {
         duration
       });
       this._handle = handle;
-      this.onStart(this);
```

Comment:
I like this change. (It simplifies the implementation of the same `onStart` event for attribute spring transitions.) Would it be crazy to move the `this._handle = timeline.addChannel({...})` up to the `start()` method as well? If we did, we could drop all the `this._handle = null`s, which would no longer be needed to trigger the creation of a new timeline channel in this `update()` method.